api/src/test/java/org/pmiops/workbench/workspaces/WorkspacesControllerTest.java

```diff
@@ -10,16 +10,7 @@
 import static org.mockito.ArgumentMatchers.anyMap;
 import static org.mockito.ArgumentMatchers.argThat;
 import static org.mockito.ArgumentMatchers.eq;
-import static org.mockito.Mockito.any;
-import static org.mockito.Mockito.anyString;
-import static org.mockito.Mockito.doReturn;
-import static org.mockito.Mockito.doThrow;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.never;
-import static org.mockito.Mockito.times;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.verifyZeroInteractions;
-import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.*;
```

Comment:
Pls undo this change (& update IntelliJ settings to avoid star-imports... which reminds me, I'd still like to package a "AoU IntelliJ settings" file to make it easier for us to have a canonical set of configs we can all rely on)

Reply:
I thought Mockito was whitelisted.